### PR TITLE
Fix some typos in code and comments

### PR DESCRIPTION
**Pull request by storagezhang**
Fix some typos while reading the lib.

One in `src/tracker/state.rs`, **ProgressState::Snapshot => write!** should be **StateSnapshot**.

```rust
impl Display for ProgressState {
    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
        match self {
            ProgressState::Probe => write!(f, "StateProbe"),
            ProgressState::Replicate => write!(f, "StateReplicate"),
            ProgressState::Snapshot => write!(f, "StateSnaphot"),
        }
    }
}
```

Others are some typos in comments.

### CODE REVIEW

**Review by BusyJay — DISMISSED** [review]
Thanks!

File: src/util.rs [hunk 1]
```diff
@@ -73,7 +73,7 @@ pub fn limit_size<T: PbMessage + Clone>(entries: &mut Vec<T>, max: Option<u64>)
 }
 
 /// Check whether the entry is continuous to the message.
-/// i.e msg's next entry index should be equal to the first entries's index
+/// i.e msg's next entry index should be equal to the first entries' index
```
Comment (NingLin-P) [inline src/util.rs:76]:
```suggestion
/// i.e msg's next entry index should be equal to the index of the first entry in `ents`
```

**Review by NingLin-P — COMMENTED** [review]
Rest LGTM, thank you!

**Comment by storagezhang** [comment]
@NingLin-P Finished!

**Review by NingLin-P — APPROVED** [review]

**Comment by storagezhang** [comment]
@BusyJay @NingLin-P

**Comment by storagezhang** [comment]
@BusyJay @NingLin-P 

Sorry to bother you, but I can't quite understand this.  

Why **The Travis CI build failed** ?

I just modify comments, and int Job log it's shown: 

```
error: custom inner attributes are unstable
 --> /home/travis/build/tikv/raft-rs/target/debug/build/raft-proto-bab3413823f45325/out/protos/eraftpb.rs:9:4
  |
9 | #![rustfmt::skip]
  |    ^^^^^^^^^^^^^
  |
  = note: `#[deny(soft_unstable)]` on by default
  = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
  = note: for more information, see issue #64266 <https://github.com/rust-lang/rust/issues/64266>
error: aborting due to previous error
error: could not compile `raft-proto`
To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
The command "cargo test --all -- --nocapture" exited with 101.
```

I don't know what's wrong.

Thanks!

**Comment by storagezhang** [comment]
I recommitted and try to check for errors but failed.

The Travis CI still failed in **Rust: nightly**.

**Review by BusyJay — APPROVED** [review]
It's unrelated to your changes.

**Pull request merged**